recipes/opus/all/FindOPUS.cmake

```diff
@@ -0,0 +1,18 @@
+find_path(
```

Comment:
Why do you need this?

Reply:
I don't know, I took bincrafters' recipe "as is". I'll do some cleanup